docs/source/database.rst

```diff
@@ -52,18 +52,18 @@ Usage
 Reasons of Failure
 ^^^^^^^^^^^^^^^^^^
 
-Possible scenarios where database is not generated
+Possible scenarios where database is not generated:
 
   * There does not exist at-least one part in the test.
-  * Any part which has started does not end before another part starts or the code ends i.e. *RVTEST_CASE_START* exists for that part but *RVTEST_CASE_END* doesn't.
-  * The part names given in a *RVTEST_CASE_START*-*RVTEST_CASE_END* pair doesn't match.
-  * *RVTEST_ISA* macro isn't present in the test.
+  * Any part which has started does not end before another part starts or the code ends i.e. ``RVTEST_CASE_START`` exists for that part but ``RVTEST_CASE_END`` doesn't.
+  * The part names given in a ``RVTEST_CASE_START``-``RVTEST_CASE_END`` pair doesn't match.
+  * ``RVTEST_ISA`` macro isn't present in the test.
 
 Notes
 ^^^^^
 
-1. The database is always alphabetically ordered
+1. The database is always alphabetically ordered.
 2. The database checks for macro sanity - i.e. certain macros exists and in the correct order.
-3. Each time a new test is added to the ``suite`` directory, the database utility has to be run manually and the database.yaml
+3. Each time a new test is added to the ``suite`` directory, the database utility has to be run manually and the ``database.yaml``
```

Comment:
This particular entry in the notes can be removed as the database is generated every time for each riscof run. It can probably be modified to:
```
3. The database file is generated by default every time for any relevant command unless a ``--database`` argument is supplied in the command. 
4. Each time a new test is added to the ``suite`` directory, the database utility has to be run  to ensure that the test is included in the ``database.yaml``. This can be done by running the `gendb` command or by skipping the ``--database`` argument in the CLI for relevant commands. 
```

Reply:
I will have a look at it this weekend.

Otherwise I still have some documentation files to go through, before the changes would be ready for a merge.

I have split the changes into multiple commits, so they are easier to review.
Every find/replace (for capitalization) has its own commit, and I will try to split remaining changes into:
- fixed width font,
- spelling and grammar,
- functionality.

I did not make changes to the Python source code, although there are many cases with missing capitalization.
Consequentially I also did not make changes to code blocks in the documentation.